config/routes.rb

```diff
@@ -1,8 +1,15 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+
+  get 'sessions/create'
+  get 'sessions/destroy'
   root 'static_pages#home'
   get 'static_pages/help'
   get "/signup",  to: "users#new"
   post "/signup", to:"users#create"
-  resources :users, only: %i(new create show)
+  resources :users, only: %i(show new create)
+
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
 end
```

Comment:
Add the `end` keyword at the end of file

Reply:
![Screenshot from 2021-09-08 11-19-54](https://user-images.githubusercontent.com/87749562/132441145-b8187893-82fa-4846-a8e9-19ec79d09f1e.png)

i already have